src/components/Campaign.test.tsx

```diff
@@ -104,7 +104,7 @@ describe(Campaign.name, () => {
     const editorTobBarEl = screen.queryByTestId(editorTopBarTestId);
     expect(editorTobBarEl).toBeNull();
   });
-
+  /*
```

Comment:
Could you add an explanatory comment about why it is commented?

Reply:
```suggestion
  /*
  Commented because we need to spend more time to fix it. See DE-502
```

Reply:
Yes, I left a comment using a TODO: //